MarkBind/markbind
# Fix minor UI issues with default.md

**What is the purpose of this pull request?**




**Overview of changes:**

Addresses two of the issues related to `_markbind/default.md` brought up in #1866.

Specific improvements:
- [X] Change the sidebar name from "Template" to "Contents" in `default.md`
- [x] Remove generation of blank lines after the navbar in `default.md`

**Anything you'd like to highlight/discuss:**


**Testing instructions:**
Try `markbind init` or `markbind init --convert` on a pre-existing set of documents, and ensure that changes are reflected.



**Proposed commit message: (wrap lines at 72 characters)**
Fix minor UI issues with default.md


---

**Checklist:** :ballot_box_with_check:
